SpatialGDK/Documentation/content/get-started/tutorial.md

```diff
@@ -47,19 +46,29 @@ In this step, you're going to build the Unreal GDK's dependencies.
+_Image: On the GDK toolbar in the Unreal Editor select **Snapshot**_<br/>
 
 ### Deploy the project locally
 
 In this section you’ll run a [local deployment](https://docs.improbable.io/reference/latest/shared/glossary#local-deployment) of the project. As the name suggests, local deployments run on your development machine (you will [cloud deploy](https://docs.improbable.io/reference/latest/shared/glossary#cloud-deployment) later in this tutorial).
 
 1. In a text editor, open `UnrealGDKThirdPersonShooter\spatial\default_launch.json` and in the `layer_configurations` section set `rows` to `2`. This instructs SpatialOS that you will be launching two server-workers.
-1. In the Unreal Editor, in the Unreal toolbar, open the **Play** drop-down menu.
-1. Under **Multiplayer Options**, enter the number of players as **2**
-1. Enter the number of servers as **2**
-1. Check the box next to **Run Dedicated Server**
-1. In Unreal Editor, in the SpatialOS GDK toolbar, select **Start**, the green play icon. This will open a terminal window and run the [`spatial local launch`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-local-launch#spatial-local-launch) command, which starts the [SpatialOS Runtime](https://docs.improbable.io/reference/latest/shared/glossary#the-runtime). It's ready when you see `SpatialOS ready. Access the inspector at http://localhost:21000/inspector`.
-1. From the Unreal Editor toolbar, click **Play** to run the game. This starts two headless server-workers and two [client-workers](https://docs.improbable.io/reference/latest/shared/glossary#client-worker).
+1. In the Unreal Editor, in the Unreal toolbar, open the **Play** drop-down menu.<br/>
+2. Under **Multiplayer Options**, enter the number of players as **2**
+3. Enter the number of servers as **2**
+4. Ensure the box next to **Run Dedicated Server** is checked<br/>
+![]({{assetRoot}}assets/set-up-template/template-multiplayer-options.png)<br/>
+_Image: The Unreal Engine **Play** drop-down menu, with **Multiplayer Options** and **New Editor Window (PIE)** highlighted_<br/>
+1. In Unreal Editor, in the SpatialOS GDK toolbar, select **Start**, the green play icon. This will open a terminal window and run the [`spatial local launch`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-local-launch#spatial-local-launch) command, which starts the [SpatialOS Runtime](https://docs.improbable.io/reference/latest/shared/glossary#the-runtime).<br/>
+![Toolbar]({{assetRoot}}assets/set-up-template/template-start.png)<br/>
+_Image: On the GDK toolbar in the Unreal Editor select **Start**_<br/>
+6. It's ready when you see `SpatialOS ready. Access the inspector at http://localhost:21000/inspector`.
+7. From the Unreal Editor toolbar, click **Play** to run the game. This starts two headless server-workers and two [client-workers](https://docs.improbable.io/reference/latest/shared/glossary#client-worker) locally.
```

Comment:
I've not seen the term 'headless server-worker' anywhere else in the docs. One for @ElleEss maybe? Think we might need to explain what we mean by headless.

Reply:
I think "dedicated server" is the more common term for Unreal.

Reply:
Thanks both
Will defer to @ElleEss

Reply:
I have never heard of this. 
What does it mean? Who wrote it? I have a very bright light I can shine in their face to interrogate.

Reply:
Also, `select **Play**` is the MMoS standard (but that's less of an issue).

Reply:
Not sure who introduced it here but we've used the term before. Spatialos runs game engines in headless mode, described for instance here https://docs.improbable.io/reference/13.6/shared/byoe/introduction#requirements-for-engine-integration